README.md

```diff
@@ -33,6 +33,20 @@ Windows: `certutil -encodehex -f release.keystore encoded.txt 0x40000001`
 4. Create a third variable SECRET_RELEASE_KEYSTORE_PASSWORD as type variable with the password of your keystore as value.
 5. Use the `android` job example in the `gitlab-ci.yml` file.
 
+### GDNative/C++
+
+See [this repository](https://github.com/2shady4u/godot-cpp-ci) for automating GDNative C++ compilation, which is based off this repository.
+
+### Modules
+
+You have to compile the modules with godot first, see [this excellent repository](https://gitlab.com/Calinou/godot-builds-ci) by Calinou for automating godot builds. 
+
+After that you would use the custom build to export your project as usual, see [this guide](https://gitlab.com/greenfox/godot-build-automation/-/blob/master/advanced_topics.md#using-a-custom-build-of-godot) by Greenfox on how to use a custom godot build for automated exports.
+
+### iOS
+
+Not available yet. Automating xcode projects is doable but not trivial, and mac runners only recently became available for github actions, so it will happen eventually.
```

Comment:
```suggestion
Not available yet. Automating Xcode projects is doable but not trivial, and macOS runners only recently became available for GitHub actions, so it will happen eventually.
```